web/components/Canvas/Board/Cards/Cards.jsx

```diff
@@ -27,13 +27,13 @@ function Cards(props) {
     gameState.game_phase === 1 && (
       <div className={styles.Cards}>
         {cards.map((name) => (
-          <CardActionArea disabled={!props?.isClickable && !gameState?.suggestion_valid_cards?.includes(name)}
```

Comment:
We don't need the second half of these conditions anymore because only valid suggestion cards are shown